lighthouse-core/test/report/html/renderer/pwa-category-renderer-test.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest, browser */
+
+const assert = require('assert');
+const fs = require('fs');
+const jsdom = require('jsdom');
+const Util = require('../../../../report/html/renderer/util.js');
+const URL = require('../../../../lib/url-shim.js');
+const DOM = require('../../../../report/html/renderer/dom.js');
+const DetailsRenderer = require('../../../../report/html/renderer/details-renderer.js');
+const CategoryRenderer = require('../../../../report/html/renderer/category-renderer.js');
+const sampleResultsOrig = require('../../../results/sample_v2.json');
+
+const TEMPLATE_FILE = fs.readFileSync(__dirname +
+    '/../../../../report/html/templates.html', 'utf8');
+
+describe('PwaCategoryRenderer', () => {
+  let category;
+  let pwaRenderer;
+  let sampleResults;
+
+  beforeAll(() => {
+    global.URL = URL;
+    global.Util = Util;
+    global.CategoryRenderer = CategoryRenderer;
+
+    const PwaCategoryRenderer =
+        require('../../../../report/html/renderer/pwa-category-renderer.js');
+
+    const {document} = new jsdom.JSDOM(TEMPLATE_FILE).window;
+    const dom = new DOM(document);
+    const detailsRenderer = new DetailsRenderer(dom);
+    pwaRenderer = new PwaCategoryRenderer(dom, detailsRenderer);
+
+    sampleResults = Util.prepareReportResult(sampleResultsOrig);
+    category = sampleResults.reportCategories.find(cat => cat.id === 'pwa');
+  });
+
+  afterAll(() => {
+    global.URL = undefined;
```

Comment:
FWIW, I'm not so sure we need to care about this now with jest

Reply:
> I think we've had this in a comment discussion before 😆

ha, yes. We should actually verify :)